llvm/lib/Transforms/Utils/InlineFunction.cpp

```diff
@@ -1355,14 +1355,23 @@ static AttrBuilder IdentifyValidAttributes(CallBase &CB) {
     Valid.addDereferenceableOrNullAttr(DerefOrNullBytes);
   if (CB.hasRetAttr(Attribute::NoAlias))
     Valid.addAttribute(Attribute::NoAlias);
+  return Valid;
+}
+
+static AttrBuilder IdentifyValidPoisonGeneratingAttributes(CallBase &CB) {
```

Comment:
We should rename the above function to ValidUBGeneratingAttributes for symmetry.

Reply:
Sorry, this was unclear because GitHub doesn't allow commenting on unchanged lines: I meant that IdentifyValidAttributes should become IdentifyValidUBGeneratingAttributes and this one should stay as IdentifyValidPoisonGeneratingAttributes.